rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import babel from 'rollup-plugin-babel';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+import pkg from './package.json';
+
+const input = './src/index.js';
+
+const external = id => !id.startsWith('\0') && !id.startsWith('/') && !id.startsWith('.');
```

Comment:
I think this makes sense, but can you add a comment to what this covers? Namely not sure what case the first statement is used for.

Reply:
Done